### ISSUE TITLE
Change Wiki main title on Anglo wiki

### ISSUE THREAD

**Issue by isabvllv** [LeHubca/lehub-mediawiki#13]
Hi Albert,

Hope you are doing well! I just got a request from our anglo coordinator about changing the main title from "Le HUB" to "The Climate Justice Organizing HUB" on the anglo server only, is it possible? https://en.wiki.lehub.ca/index.php/Main_Page 

Otherwise, we could have it named : Le Hub/The Climate Justice Organizing HUB ?

Resource : https://www.mediawiki.org/wiki/Change_the_name_of_the_wiki

**Comment by alberto56** [comment]
This needed to be done in code (there is no way to do it from the admin interface). I have modified the following lines:

$wgSitename = "Le Hub/The Climate Justice Organizing HUB";
$wgMetaNamespace = "Le Hub/The Climate Justice Organizing HUB";

in fr.wiki.lehub.ca:/root/starterkit-mediawiki-en/LocalSettings.php

I ran this and I'm still getting the old title, even after a page cache purge. I will continue investigating.
